wasm/src/api.rs

```diff
@@ -1,24 +1,40 @@
+use std::rc::Rc;
 use wasm_bindgen::prelude::*;
+use js_sys::{Array};
 
+use crate::lazy_node::LazyNode;
 use clvmr::allocator::Allocator;
 use clvmr::chia_dialect::ChiaDialect;
+use clvmr::chia_dialect::NO_NEG_DIV as _no_neg_div;
+use clvmr::chia_dialect::NO_UNKNOWN_OPS as _no_unknown_ops;
 use clvmr::cost::Cost;
 use clvmr::node::Node;
 use clvmr::run_program::run_program;
-use clvmr::serialize::{node_from_bytes, node_to_bytes};
+use clvmr::serialize::{node_from_bytes, node_to_bytes, serialized_length_from_bytes};
 
 // When the `wee_alloc` feature is enabled, use `wee_alloc` as the global
 // allocator.
 #[cfg(feature = "wee_alloc")]
 #[global_allocator]
 static ALLOC: wee_alloc::WeeAlloc = wee_alloc::WeeAlloc::INIT;
 
-// NOTE: This is just a proof of concept.
-// Ideally, the wasm api will have more features, like the
-// python api. For now, this is just a sanity check that something
-// works at all.
-//
-// TODO: replace the below with something more robust and feature-filled
+#[wasm_bindgen]
+pub struct Flag;
+
+#[wasm_bindgen]
+impl Flag {
+    #[wasm_bindgen]
+    pub fn no_neg_div() -> u32 { _no_neg_div }
+
+    #[wasm_bindgen]
+    pub fn no_unknown_ops() -> u32 { _no_unknown_ops }
+}
+
+#[wasm_bindgen]
+pub fn serialized_length(program: &[u8]) -> u64
+{
+    serialized_length_from_bytes(program).unwrap()
```

Comment:
if `program` has invalid serialization, this will take down the whole process. that seems risky, rather than propagating the error

Reply:
@arvidn I've updated the code as you requested. Please confirm when you have time.